Applications/Resnet/tensorflow/resnet18.py

```diff
@@ -122,27 +136,27 @@ def write_fn(*items, filename='./resnet.bin'):
         # print(layer.name)
         initial_weights += layer.weights.copy()
 
-    write_fn(*initial_weights, './resnet_initial.bin')
+    write_fn(*initial_weights, "./resnet_initial.bin")
 
     if DEBUG:
-        input = tf.reshape(tf.convert_to_tensor(x_train[0]), [1,3,32,32])
+        input = tf.reshape(tf.convert_to_tensor(x_train[0]), [1, 3, 32, 32])
 
         # run with training mode
         output = model.call(input)
         # run each layer with training mode
         K.set_learning_phase(1)
         for layer in model.layers:
             l_out = K.function(model.layers[0].input, layer.output)([input])
-            if (len(l_out.shape) > 2):
+            if len(l_out.shape) > 2:
                 print(layer.name, l_out[0][0][0])
             else:
                 print(layer.name, l_out[0])
 
     model.fit(
-        x = x_train,
-        y = y_train,
-        epochs = 60,
-        verbose = 1,
+        x=x_train,
+        y=y_train,
+        epochs=60,
+        verbose=1,
```

Comment:
There are so many changes only relasted with fomatter.
I'm not sure that only formatting chages are needed.

Reply:
I will restore formatting changes